Utils/gitlab_triggers/trigger_demisto_sdk_release.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+# This script triggers Demisto SDK release job in gitlab-CI.
+
+# For this script to work you will need to use a trigger token (see here for more about that: https://docs.gitlab.com/ee/ci/triggers/#create-a-pipeline-trigger-token)
+
+# This script requires the gitlab-ci trigger token and release version.
+# The branch to run against is an optional parameter(the default is master branch).
+# The Slack channel to send messages to is an optional parameter(the default is dmst-sdk-release).
+
+# Ways to run this script are:
+# trigger_demisto_sdk_release.sh -ct <trigger-token> -rv <release-version> [-b <branch-name> -ch <slack-channel-name>]
+if [ "$#" -lt "1" ]; then
+  echo "Usage:
+  $0 -ct <token>
+
+  [-ct, --ci-token]      The ci gitlab trigger token.
+  [-rv, --release-version]      The release version.
+  [-r, --reviewer]         Github username of the release owner.
+  [-ch, --slack-channel] A Slack channel to send notifications to. Default is dmst-sdk-release.
+  [-b, --branch]         The content branch name to run the .gitlab-ci.sdk-release.yml workflow. Default is master branch.
+  [-d, --is-draft]         Whether to create draft release and draft pull requests or not. Default is FALSE.
+  [-s, --sdk-branch-name]         From which branch in demisto-sdk we want to create the release. Default is master.
+  "
+  echo "Get the trigger token from here https://vault.paloaltonetworks.local/home#R2VuZXJpY1NlY3JldERldGFpbHM6RGF0YVZhdWx0OmIyMzJiNDU0LWEzOWMtNGY5YS1hMTY1LTQ4YjRlYzM1OTUxMzpSZWNvcmRJbmRleDowOklzVHJ1bmNhdGVk" # disable-secrets-detection  TODO
```

Comment:
refer to the confluence page on SDK release, and not to the vault, and add the vault url to the confluence page